client/package.json

```diff
@@ -7,13 +7,16 @@
     "aave-js": "^1.2.1",
     "bootstrap": "^4.6.0",
     "fortmatic": "^2.2.1",
+    "moment": "^2.29.1",
     "react": "16.11.0",
     "react-bootstrap": "^1.4.3",
     "react-dom": "16.11.0",
+    "react-icons": "^4.1.0",
     "react-particles-js": "^3.4.1",
     "react-router-bootstrap": "^0.25.0",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.2.0",
+    "react-share": "^4.3.1",
```

Comment:
We deleted the client/package-lock.json with the switch to yarn, so we'll want to update yarn.lock instead

Can we:
1. Delete `client/package-lock.json`
2. Install these using yarn: `yarn workspace client add --save react-icons react-share` which will update yarn.lock
3. Remove the `moment` if it's not being used